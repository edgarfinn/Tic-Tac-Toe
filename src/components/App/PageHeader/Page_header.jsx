import React from 'react';
import styled from 'styled-components';

export default () => {
  const Header = styled.header`
    background: inherit;
    position: relative;
    overflow: hidden;
    padding: 1em;
  `;

  const Title = styled.h1`
    font-family: 'Permanent Marker';
    color: #fc8aa6;
    font-size: 3em;

    @media only screen and (min-width: 735px) {
      font-size: 4em;
    }
  `;

  return (
    <Header>
      <Title>Tic Tac Toe</Title>
    </Header>
  );
};
