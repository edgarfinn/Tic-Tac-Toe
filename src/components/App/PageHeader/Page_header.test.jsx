/* eslint-env jest */
import React from 'react'
import Adapter from 'enzyme-adapter-react-16'
import Enzyme, { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import PageHeader from './Page_header'

Enzyme.configure({ adapter: new Adapter() })

const setup = () => {
  const wrapper = shallow(<PageHeader />)
  return { wrapper }
}

describe('<PageHeader />', () => {
  const { wrapper } = setup()
  it('renders as expected', () => {
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
