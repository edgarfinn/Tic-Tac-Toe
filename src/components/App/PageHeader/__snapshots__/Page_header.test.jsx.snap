// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageHeader /> renders as expected 1`] = `
<styled.header>
  <styled.h1>
    Tic Tac Toe
  </styled.h1>
</styled.header>
`;
